src/main.lua

```diff
@@ -81,6 +81,7 @@ local ActionMap = {
 	DelegateStake = "Delegate-Stake",
 	DecreaseDelegateStake = "Decrease-Delegate-Stake",
 	CancelDelegateWithdrawal = "Cancel-Delegate-Withdrawal",
+	InstantDelegateWithdrawal = "Instant-Delegate-Withdrawal",
```

Comment:
We have this already.

InstantDelegateWithdrawal is what a delegate would run to instantly unlock tokens that were already in a vault.  It accepts a gateway and a vault ID.

DecreaseDelegateStake with the `instant=true` tag would instantly unlock tokens for the max penalty.

If we were to combine DecreaseDelegateStake with InsteadDelegateWithdrawl, we need to also pass a vault ID.  You think its worthwhile to still combine these even though they have different impacts?  We split out CancelDelegateWithdrawal instead of adding that to DecreaseDelegateStake, which is what I modeled InstantDelegateWIthdrawal after.

Reply:
ah i see - thanks for the clarification. in that case, it makes sense to leave them as separate